### PR TITLE
Product models

### PR DESCRIPTION
**Pull request by alexandrelissowski**

### CODE REVIEW

**Comment by alexandrelissowski** [comment]
English request
name: Alexandre Lissowski
size: L
priority: Minor

**Comment by nestorAkeneo** [comment]
Your proofreading request has been created

Ticket link: https://akeneo.atlassian.net/browse/ENG-31:

File: content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md [hunk 1]
```diff
@@ -28,13 +28,18 @@ Only the attributes marked as 'required' are taken into account for the complete
 :::
 
 ::: warning
-Custom attributes are not supported by the Onboarder and suppliers are not able to enrich data of custom attributes. 
+Custom attributes are not supported by the Onboarder and suppliers are not able to enrich data of custom attributes.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md:31]:
```suggestion
Custom attributes are not supported by the Onboarder and suppliers are not able to enrich custom attributes data.
```

File: content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md [hunk 2]
```diff
@@ -28,13 +28,18 @@ Only the attributes marked as 'required' are taken into account for the complete
...
 :::
 
-# Via flat files import
+::: warning
+When you [create a supplier](/onboarder/articles/create-supplier.html), you can chose to allow the product creation.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md:35]:
```suggestion
When you [create a supplier](/onboarder/articles/create-supplier.html), you can choose to allow them to create products.
```

File: content/md/retailer-onboarder/define-suppliers-scope/define-product-supplier.md [hunk 3]
```diff
@@ -35,3 +35,7 @@ It's really easy to assign a supplier to a product:
 1. Click on the dropdown
 1. Select the supplier name you want to add
 1. Click on `Save`
+
+:::warning
+When you link a product variant to a supplier, the supplier can also enrich the product model associated.Then, you will [receive proposals](onboarder/articles/review-product-changes.html) when product variants are completed.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/define-suppliers-scope/define-product-supplier.md:40]:
```suggestion
When you link a product variant to a supplier, the supplier can also enrich the product model associated. Then, you will [receive proposals](onboarder/articles/review-product-changes.html) when product variants are completed.
```

File: content/md/retailer-onboarder/first-steps/synchronization.md [hunk 4]
```diff
@@ -28,10 +28,17 @@ As a consequence, the family, the attribute groups, attributes and attribute opt
...
-
+# Products synchronization
+## Products
 As soon as you link a product to a supplier and you set a supplier reference for a product, the attributes mentioned above will be synchronized with the relevant Supplier Onboarder. Therefore, the product will be displayed in the relevant supplier Onboarder product grid.
 
+## Product Models and variants
+To link a product variant to a supplier, you need to complete the supplier reference and the supplier attribute as explained for products. The parent product model is sent alongside with the product variant. You need to repeat the operation for each product variant of a product model to link variants to one or several suppliers.
+
+::: warning
+When you link 2 variants of the same product model to 2 different suppliers, both suppliers can enrich the product model and they can send you proposals.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/first-steps/synchronization.md:39]:
```suggestion
When you link 2 different variants of the same product model to 2 different suppliers, both suppliers can enrich the product model and they can send you proposals.
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 5]
```diff
@@ -6,28 +6,27 @@ popular: false
 ---
 
 # How to manage suppliers proposals?
-
-As soon as a product will be completed on the Supplier Onboarder (aka all the attributes you defined as required for the channel Onboarder are filled in by the supplier) it will be sent to your Retailer Onboarder so that you can review it.
+## Products
+When product is completed on the Supplier Onboarder (ie [all attributes defined as required for the channel Onboarder](/onboarder/articles/define-product-requirements.html) are filled in by the supplier), it is sent as a proposal to your PIM.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:10]:
```suggestion
When a product is completed on the Supplier Onboarder (ie [all attributes defined as required for the channel Onboarder](/onboarder/articles/define-product-requirements.html) are filled in by the supplier), you receive a proposal in your PIM.
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 6]
```diff
@@ -6,28 +6,27 @@ popular: false
...
-1. Click on `Product data`
-1. Then on `Onboarder proposals`
-
-The proposals provided by your suppliers will be displayed.
+You can review the suggestions made by suppliers in the [PIM proposals](https://help.akeneo.com/pim/serenity/articles/review-products-proposals.html).
 
-![Proposals list](../img/RETAILER_Proposals.png)
+## Product models with variants
+[Product models with variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html) are specific type of products you can receive proposals for.
+You can receive proposals in two different cases :
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:20]:
```suggestion
You can receive proposals in two different cases:
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 7]
```diff
@@ -6,28 +6,27 @@ popular: false
...
+1. for the product model values
+1. for the product variant values
 
-If you are not yet familiar with the proposals review, you can learn in [this article](https://help.akeneo.com/articles/review-products-proposals.html#review-actions-on-proposals) how to work with them.
+For the product model values, you receive a proposal:
+* the first time the supplier completes a product variant with its parents product models,
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:25]:
```suggestion
* the first time the supplier completes a product variant with its parent product model,
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 8]
```diff
@@ -6,28 +6,27 @@ popular: false
...
+* and then each time a product model is updated.
 
-:::info
-This screen will only display the proposals from your suppliers in their Onboarder. The proposals provided in the PIM are still displayed in `Activity`/ `Proposals`.
-:::
+For the product variant values, you receive a proposal each time a supplier complete or update a product variant.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:28]:
```suggestion
For the product variant values, you receive a proposal each time a supplier completes or updates a product variant.
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 9]
```diff
@@ -6,28 +6,27 @@ popular: false
...
 
-:::info
-New assets suggested by suppliers are included in the proposals and they are automatically added to the right asset collection when the new product is accepted. You can [change the asset](https://help.akeneo.com/pim/serenity/articles/manage-asset-families.html)(label, other attributes).
-:::
+## Assets
+New assets suggested by suppliers are included in the proposals and they are automatically added to the right asset collection when the new product is accepted.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:31]:
```suggestion
When new assets are suggested by suppliers, they are included in the proposals and automatically added to the right asset collection when the new product is accepted.
```

File: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md [hunk 10]
```diff
@@ -6,28 +6,27 @@ popular: false
...
+You can [change the asset](https://help.akeneo.com/pim/serenity/articles/manage-asset-families.html) (label, other attributes).
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md:32]:
```suggestion

```

File: content/md/supplier-onboarder/update-products.md [hunk 11]
```diff
@@ -6,49 +6,68 @@ popular: true
...
 The header panel shows the following product information:
 * The product name
 * An image of the product (if available)
+* Completeness
+* Breadcrumb to navigate within the [product model and variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html)
+* Number of missing attributes
+* Filters to access directly to attribute groups or missing attributes
+
+:::info
+The breadcrumb is displayed only when you are on a product model page, it allows you to navigate through the levels of the product model.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/update-products.md:32]:
```suggestion
The breadcrumb is displayed only when you are on a product model page, it allows you to navigate through the different levels of the product model.
```

File: content/md/supplier-onboarder/update-products.md [hunk 12]
```diff
@@ -6,49 +6,68 @@ popular: true
...
+:::
 
 ## The product navigation panel
 
+![Navigation panel highlight](../img/SUPPLIER-pef-navigation.png)
+
 The product navigation panel allows you to switch between different sections of your product:
-* Attributes: Displays the list of attributes of a product. This section is used to enrich your product. This is the most used view.
+* [Attributes](/onboarder/articles/update-products.html#the-attribute-panel): Displays the list of attributes of a product. This section is used to enrich your product. This is the most used view.
+* Assets: Allows you to upload media files when required. This section only shows if you can add assets to the product.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/update-products.md:41]:
```suggestion
* Assets: Allows you to upload media files when required. This section is only displayed if assets are required to complete the product.
```

File: content/md/supplier-onboarder/update-products.md [hunk 13]
```diff
@@ -6,49 +6,68 @@ popular: true
...
-:::info
-All the attributes that are displayed on the product form have been defined by the retailer.
-:::
+The attribute panel displays all the attributes you can enrich in the Onboarder. This list is defined by the retailer.
 
+### Completeness
 All the attributes marked with the yellow bullet point are required for the retailer.
 
-If the attribute doesn't have the bullet point, it means that the retailer make it available for the supplier to complete but it is more a nice to have to help the retailer. 
+The attributes without the bullet point are optional and are not taken in account for the completeness calculation. When you complete these attributes, you help the retailer to get better product values.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/update-products.md:62]:
```suggestion
The attributes without the bullet point are optional and are not taken into account for the completeness calculation. When you complete these attributes, you help the retailer to get better product values.
```

**Review by camillebourjade — COMMENTED** [review]
English fix

**Pull request merged**